content/en/docs/releases/supported-releases/index.md

```diff
@@ -55,8 +55,8 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 |-----------------|-----------------------|---------------------|----------------------|-------------------------------|-----------------------------------------------|
 | master          | No, development only  |                     |                      |                               |                                               |
 | 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17                                    |
-| 1.9             | Yes                   | February 9, 2021    | ~Aug 2021 (Expected) | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
-| 1.8             | Yes                   | November 10, 2020   | ~May 2021 (Expected) | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
+| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
+| 1.8             | Yes                   | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
 | 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
 | 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
```

Comment:
How long do we keep these rows in the table for?

Reply:
We don't remove them. There are many users on older versions of the software that we believe haven't upgraded and either don't know or care to upgrade. More visibility is good in this case so that people will notice and upgrade their existing systems.